docs/create-gtid-replica.md

```diff
@@ -112,9 +91,6 @@ If you are using a version 8.0.23 or later, run the following commands:
  > START REPLICA;
 ```
 
-If you are using version 8.0.23 or later,
-use [CHANGE_REPLICATION_SOURCE_TO and the appropriate options](https://dev.mysql.com/doc/refman/8.0/en/change-replication-source-to.html). `CHANGE_MASTER_TO` is deprecated as of that release.
-
 !!! note
```

Comment:
Move note above the example. Should not tell someone something important after.

Reply:
fixed